db/migrate/20170718022731_remove_call_number_from_collection.rb

```diff
@@ -0,0 +1,9 @@
+class RemoveCallNumberFromCollection < ActiveRecord::Migration
+  def up
```

Comment:
you can make this migration auto-reversible with just a `change` method rather than `up`/`down`, but that's just a convenience thing to do going forward rather than change this one